tls/tls_ticket.c

```diff
@@ -533,31 +534,22 @@ ttls_ticket_sess_load(TlsState *state, size_t len, unsigned long lifetime)
 		if (!sess->peer_cert)
 			return TTLS_ERR_ALLOC_FAILED;
 		/*
-		 * Same as in See ttls_parse_certificate(), we have to copy full
+		 * Same as in ttls_x509_crt_parse(), we have to copy full
 		 * certificate here, since some structures in TlsX509Crt may
 		 * address it.
 		 */
-		pg = alloc_pages(GFP_ATOMIC, get_order(state->cert_len));
+		sess->peer_cert->raw.order = get_order(state->cert_len + TTLS_CERT_LEN_LEN);
```

Comment:
I must pay the attention. This branch of the code is completely disabled, above we have a `BUG()`.